Source/WebCore/rendering/RenderSearchField.cpp

```diff
@@ -310,16 +311,19 @@ int RenderSearchField::clientInsetRight() const
 LayoutUnit RenderSearchField::clientPaddingLeft() const
 {
     LayoutUnit padding = paddingLeft();
-    if (RenderBox* box = innerBlockElement() ? innerBlockElement()->renderBox() : 0)
+    RefPtr innerBlock = innerBlockElement();
+    if (RenderBox* box = innerBlock ? innerBlock->renderBox() : 0)
         padding += box->x();
     return padding;
 }
 
 LayoutUnit RenderSearchField::clientPaddingRight() const
 {
     LayoutUnit padding = paddingRight();
-    if (RenderBox* containerBox = containerElement() ? containerElement()->renderBox() : 0) {
-        if (RenderBox* innerBlockBox = innerBlockElement() ? innerBlockElement()->renderBox() : 0)
+    RefPtr container = containerElement();
+    RefPtr innerBlock = innerBlockElement();
```

Comment:
This should be moved inside the `if` scope, no reason to ref this if we don't end up using it.

Reply:
Don't we need to have them exist in order to execute the if statement?

Reply:
Is this clearer?
```
RefPtr container = containerElement();
if (auto* containerBox = container ? container->renderBox() : nullptr) {
    RefPtr innerBlock = innerBlockElement();
    if (auto* innerBlockBox = innerBlock ? innerBlock->renderBox() : nullptr)
```